content/api/events-ingest.apib

```diff
@@ -23,7 +23,12 @@ Please see [Processing Out of Band Errors](#header-processing-out-of-band-errors
 
 ### Processing Out of Band Errors
 
-**Coming Soon**
+Every uploaded batch of events can be tracked via the `id` returned from the API after a successful POST. This batch ID can be used in conjunction with the Events API to
+view the outcome of SparkPost's pre-processing. 
+
+Note: the ingested email events may still take up to two minutes to be viewable
```

Comment:
I'm not sure that I would commit to 2 minutes here.  That very much feels like a guarantee.  I would say a `few moments` or `a short time` unless that commitment was required from someone else.